Sources/protoc-gen-swift/FileGenerator.swift

```diff
@@ -122,7 +122,7 @@ class FileGenerator {
         }
 
         p.print()
-        generateVersionCheck(printer: &p)
+        // generateVersionCheck(printer: &p)
```

Comment:
Let's delete instead of commenting it out.

Later on we can use git diff to compare changes.

Reply:
yes, updated it